app/views/explore/index.html.haml

```diff
@@ -5,6 +5,11 @@
     %h1= Figaro.env.dashboard_title
 
 .container
+  = form_tag explore_path, class: 'explore-courses', method: :get do
+    = text_field_tag(:search, '', placeholder: "Search the courses")
```

Comment:
The placeholder needs to be an i18n messsage, and it needs to switch on `@presenter.course_string_prefix` (as with the table sort controls, for example).

The two versions in the `courses` and `courses_generic` message key sections  should be "Search courses" and "Search programs".